fakturogen-web/src/main/resources/application.properties

```diff
@@ -11,4 +11,6 @@ spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.format_sql=true
 
 comarch.credentials.clientId=123
-comarch.credentials.secret=abc
\ No newline at end of file
+comarch.credentials.secret=abc
+
+comarchApiCustomerConnector.url=https://app.erpxt.pl/api2/public/customers
```

Comment:
I think, you've missed part of the link,  should be this: https://app.erpxt.pl/api2/public/v1.2/customers